kubectl-rook-ceph.sh

```diff
@@ -344,6 +433,10 @@ function run_main_command() {
 : "${ROOK_CLUSTER_NAMESPACE:=rook-ceph}"
 : "${ROOK_OPERATOR_NAMESPACE:=$ROOK_CLUSTER_NAMESPACE}"
 : "${TOP_LEVEL_COMMAND:=kubectl}"
+: "${RESET:=\033[0m}"                    # For no color
+: "${ERROR:=\033[1;31mError $RESET}"     # \033[1;31m for Red color
+: "${INFO:=\033[1;32mInfo $RESET}"       # \033[1;32m for Green color
+: "${WARNING:=\033[1;33mWarning $RESET}" # \033[1;33m for Yellow color
```

Comment:
How about we name them ERROR_PREFIX, INFO_PREFIX, and WARNING_PREFIX? 
```suggestion
: "${WARNING_PREFIX:=\033[1;33mWarning $RESET}" # \033[1;33m for Yellow color
```

Reply:
It might be good to make info blue instead of green. Green usually indicates a "good" or "correct" state.

Reply:
changed